### DISCUSSION TITLE
Web-page

### DISCUSSION THREAD

**Discussion by localadmin33** [postalserver/postal#3055] [Help with using Postal]
### Good afternoon. Using the instructions for installing Postal, I came across the fact that the web page does not open for me. I installed Caddy, but it still doesn't open the page.
My yml:
version: 2

postal:
  web_hostname: postal.filter-ru.com
  web_protocol: https
  smtp_hostname: postal.filter-ru.com

main_db:
  host: 10.0.1.111
  username: root
  password: postalpassword
  database: postal

message_db:
  host: 10.0.1.111
  username: root
  password: postalpassword
  prefix: postal

smtp_server:
  default_bind_address: "::"

dns:
  # Specify the DNS records that you have configured. Refer to the documentation at
  # https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration for further
  # information about these.
  mx_records:
    - mx.postal.filter-ru.com
  spf_include: spf.postal.filter-ru.com
  return_path_domain: rp.postal.filter-ru.com
  route_domain: routes.postal.filter-ru.com
  track_domain: track.postal.filter-ru.com

smtp:
  # Specify an SMTP server that can be used to send messages from the Postal management
  # system to users. You can configure this to use a Postal mail server once the
  # your installation has been set up.
  host: 10.0.1.111
  port: 2525
  username: # Complete when Postal is running and you can
  password: # generate the credentials within the interface.
  from_name: Postal
  from_address: postal.filter-ru.com

rails:
  # This is generated automatically by the config initialization. It should be a random
  # string unique to your installation.
  secret_key:*****

My caddyfile:

# Update this to match the hostname you wish to use for accessing the
# Postal web interface.
postal.filter-ru.com {
  reverse_proxy 127.0.0.1:5000
}

# If you use open/click tracking you may need to add additional hosts
# to this file using the format below.
#
# click.yourdomain.com {
#   reverse_proxy 127.0.0.1:5000 {
#     header_up X-Postal-Track-Host "1"
#   }
# }

**Comment by willpower232** [comment]
You should confirm Postal is working with `postal status` and if it is, that you don't have any firewalls blocking the web ports and your URL is correctly pointing at your server.
